examples/with-elysia-and-bun/main.ts

```diff
@@ -0,0 +1,21 @@
+import Bull from "bull";
+import { Elysia } from "elysia";
+import { queuedash } from "@queuedash/api";
+
+const app = new Elysia().use(
```

Comment:
@wackfx what do you think about this api? it feels more natural in the elysia ecosystem

Reply:
I believe this is great! I wasn't sure that you would accept to change it as it looked like it was the same pattern between fastify and express in the examples, but this is much better indeed.